packages/event-handler/src/rest/BaseRouter.ts

```diff
@@ -32,7 +36,7 @@ abstract class BaseRouter {
    */
   protected readonly isDev: boolean = false;
 
-  public constructor(options?: RouterOptions) {
+  protected constructor(options?: RouterOptions) {
```

Comment:
Shouldn't the constructor be `public` so the class can be instantiated? I've never seen it declared as `protected` because essentially, even when extending you can't overload it but should instead use `super()` in the child class.

Reply:
I see, I didn't know about this specific nuance in abstract classes

Reply:
Neither did I!